client/src/app/+videos/+video-edit/video-add-components/video-upload.component.ts

```diff
@@ -205,7 +205,7 @@ export class VideoUploadComponent extends VideoSend implements OnInit, OnDestroy
       this.resumableUploadService.handleFiles(this.previewfileUpload, {
         ...this.options,
         metadata: {
-          isAudioBg: true
+          isAudio: true
```

Comment:
Why rename this? We're sending the background image here, not the audio file.

Reply:
You're right. I will rename it to `previewfileUploadForAudio` or something more explicit, because `Bg` is not clear enough ; it got me confused at least --'